worker/src/crypto.rs

```diff
@@ -0,0 +1,65 @@
+use js_sys::{ArrayBuffer, Uint8Array};
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_futures::JsFuture;
+
+/// A Rust-friendly wrapper around the non-standard [crypto.DigestStream](https://developers.cloudflare.com/workers/runtime-apis/web-crypto/#constructors) API
+///
+/// Example usage:
+/// ```rust
+/// let digest_stream = DigestStream::new(DigestStreamAlgorithm::Sha256);
+///
+/// // create a ReadableStream from a string
+/// let mut req_init = web_sys::RequestInit::new();
+/// req_init.method("POST");
+/// req_init.body(Some(&JsValue::from_str("foo")));
+/// let req = web_sys::Request::new_with_str_and_init("http://internal", &req_init).unwrap();
+/// let body = req.body().unwrap();
+///
+/// // just kick the promise off to the background, we'll await the digest itself
+/// // since this is piped to a JS readable stream, call .raw() to get the underlying JS object
+/// let _ = body.pipe_to(digest_stream.raw());
+///
+/// let bytes:Vec<u8> = digest_stream.digest().await.unwrap().to_vec();
+/// ```
+pub struct DigestStream {
+    inner: worker_sys::DigestStream,
+}
+
+impl DigestStream {
+    #[allow(dead_code)]
```

Comment:
Are these actually dead? Since you publicly export this module, it should be fine.

Reply:
you're right, removed in https://github.com/cloudflare/workers-rs/pull/529/commits/0527af1f904db199415207bc2044f63c12a0b5e8 :)